gap/digraph.gi

```diff
@@ -1368,6 +1423,259 @@ function(filt, n, p)
   return DigraphNC(IsMutableDigraph, RANDOM_DIGRAPH(n, Int(p * 10000)));
 end);
 
+BindGlobal("DIGRAPHS_FillOutGraph", function(n, p, adjacencyList)
```

Comment:
It'd be good to have a comment here explaining briefly what `DIGRAPHS_FillOutGraph` is supposed to do.